test/tools/Modules/HelpersDebugger/HelpersDebugger.psm1

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Ensure that terminating errors terminate when importing the module.
+trap {throw $_}
+
+# Strict mode FTW.
+Set-StrictMode -Version Latest
+
+# Enable explicit export so that there are no surprises with commands exported from the module.
+Export-ModuleMember
```

Comment:
We have nothing to export yet, so why calling this at this point?

Reply:
I do this in every module I author. A naked invocation of `Export-ModuleMember` simply tells PowerShell to turn off default exporting of commands. I've been using this pattern with explicit export of commands in modules for longer than I can remember.

Reply:
Just curious, if you explicitly have `Export-ModuleMember -Function xxx` after every function definition in the module, do we still need this line?

Reply:
This is one of those things I've done for a long time and I haven't been asked about it so I haven't had the need to revisit the technical details behind the pattern I follow much.

I just dug into this now, and no, technically it's not necessary if you have other `Export-ModuleMember` invocations in your script module. As a pattern though, I think I still prefer it because it lets anyone looking at it know that the module is designed for explicit export.

Thanks for the good question. :)